opensearch-project/OpenSearch-Dashboards
# [Navigation-Next] Add current nav group into chrome service

### Description

1. Add current nav group into chrome service
2. prepend current nav group to breadcrumbs

### Issues Resolved




#7131, #7132

## Screenshot



![image](https://github.com/opensearch-project/OpenSearch-Dashboards/assets/113890546/834812b5-8d4b-49b2-8f8d-f8a0d8dd7012)


## Testing the changes



1. start OpenSearch Dashboards
2. go to `app/management/opensearch-dashboards/settings` to enable `Use New Home Page` (nav group share same configuration)
3. open dev tools to set current nav group `sessionStorage.setItem('core.chrome.currentNavGroupId','analytics')`
4. go to dashboard/visualization page and then refresh page 
5. you will see current nav group has been prepend to current breadcrumb

## Changelog




- feat: 1. Add current nav group into chrome service 2. Prepend current nav group into breadcrumb

### Check List
